ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/CustomerCenterErrorView.kt

```diff
@@ -0,0 +1,33 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter.views
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.PurchasesErrorCode
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.CustomerCenterState
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.theme.CustomerCenterPreviewTheme
+
```

Comment:
The only new things in `CustomerCenterErrorView` and `CustomerCenterLoadingView` are:
- Added `View` suffix to the names
- Made them `internal` instead of `private`
- Added previews

This allows us to reuse these views between the `InternalCustomerCenter` and the `VirtualCurrencyBalancesScreen` 👍